test/test_resize.cpp

```diff
@@ -1830,20 +1801,8 @@ TEST_F(ResizeTest, FusionSliceForNanoGPT2) {
       "Unexpected slice producer: ",
       known_slice_producer->toString());
 
-  auto aten_t2 = t0 + 1;
-  auto aten_slice = aten_t2.index(
-      {at::indexing::Slice(0, 32, 1), at::indexing::Slice(0, 32, 1)});
-  auto aten_t4 = aten_slice + t1;
-
-  auto aten_t7 = aten_t2 + 1;
-
   testValidate(
-      executor_cache.fusion(),
-      cg_outputs,
-      aten_inputs,
-      {aten_t4, aten_t4, aten_t7},
-      __LINE__,
-      __FILE__);
+      executor_cache.fusion(), cg_outputs, aten_inputs, __LINE__, __FILE__);
```

Comment:
I have a question for @zasdfgbnm and maybe @naoyam here. I get that using the evaluator for the ATen ground truth is useful in case we don't want to provide the manually-defined reference. However, if we're testing a chunk of code that modifies the fusion graph, we might want to keep the ATen reference since the automatic method can only check that the output is consistent with the graph itself. I am thinking mostly of the resize and dynamic shape tests where there is a danger of modifying the graph in a way that subtly changes the output even though it's self-consistent. For example if we replaced a pad with `set` in some op, the automatic tests might not catch the bug. Since we already have working references in existing tests, maybe we should just leave them, but feel free to omit for new tests? What do you guys think?

Reply:
That's a very good point. Can we use the original fusion since it should not be modified?

Reply:
Actually yes this should use the unmodified fusion `executor_cache.fusion()`. If we instead grabbed the last compiled runtime from the cache and used its complete fusion as the reference, we might get something different. I guess I hadn't considered that we're moving unique pointers around, so there's no real danger of unwittingly passing the modified fusion here.